types/react-window/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-null-undefined-union": false // needed for optional Refs
```

Comment:
Can you add a comment at the point in the code that requires this rule to be disabled?
I don't get any lint errors when running dtslint locally with your changes, so I want to make sure it's absolutely needed to disable this.

Reply:
You're right, it's not necessary. I was invoking `dtslint` incorrectly locally.

Pushed a commit to revert.